app/src/main/java/com/nutomic/syncthingandroid/activities/SettingsActivity.java

```diff
@@ -578,7 +581,10 @@ private boolean handleHttpProxyPreferenceChange(Preference preference, String ne
             if (newValue.equals("")) {
                 preference.setSummary(getString(R.string.do_not_use_proxy) + " " + getString(R.string.generic_example) + ": " + getString(R.string.http_proxy_address_example));
                 return true;
-            } else if (newValue.matches("^http://(\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}\\.\\d{1,3}):(\\d{1,5})$")) {
+            } else if (newValue.matches("^http://" +
```

Comment:
Also, I think https proxy should use a https scheme, so you might need a separate field as I suspect this might not work.

Reply:
According to https://wiki.ubuntuusers.de/Proxyserver/ https proxy url is the same as http proxy url:
[quote]
export http_proxy=http://SERVER-IP_ODER_NAME:PORT
export https_proxy=http://SERVER-IP_ODER_NAME:PORT

Reply:
It seems that there are varying opinions about this on the internet, but I guess it's ok.